tests/api_tests/nodes/test_serializers.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+from urlparse import urlparse
+
+from nose.tools import *  # flake8: noqa
+from dateutil.parser import parse as parse_date
+
+from tests.base import DbTestCase, ApiTestCase, assert_datetime_equal
+from tests.utils import make_drf_request
+from tests.factories import UserFactory, NodeFactory, RegistrationFactory, ProjectFactory
+
+from framework.auth import Auth
+from api.nodes.serializers import NodeSerializer, NodeRegistrationSerializer
+from api.base.settings.defaults import API_BASE
+
+
+class TestNodeSerializer(DbTestCase):
```

Comment:
Ha, nice. Yeah, validating against the JSON-API schema is definitely a good thing. 

These tests serve a different purpose: testing that the serialized representations of our stored objects have the correct values. For example, these tests were added to ensure that the `registration_from` URLs that I added were actually correct.